doc/specification.md

```diff
@@ -80,7 +80,7 @@ The JSON files additionally follow
 
 ## Format Compatibility
 
-The format of object types will evolve over time. If a change to the format is considered incompatible, the version of the file format has to be increased.
+The format of objects will evolve over time. If a change to the format is considered incompatible, the version of the file format has to be increased.
```

Comment:
I am not sure whether we should stick to `object type` here.

Reply:
Reverted to the previous version.